tools/gulp/packaging/build-utils.ts

```diff
@@ -0,0 +1,85 @@
+import {writeFileSync, readFileSync} from 'fs-extra';
+import {join} from 'path';
+import {LICENSE_BANNER, MATERIAL_VERSION} from '../constants';
```

Comment:
I really dislike grab-bag "util" files. Can we break this up into individual files?